slacker/commands/channels/channels_list_command.py

```diff
@@ -0,0 +1,61 @@
+from slacker.commands.command import Command
+from slacker.commands.argument_parser import ArgumentParser
+from slacker.slack_api import SlackAPI
+
+class ChannelsListCommand(Command):
+  def name(self):
+    return 'channels.list'
+
+  def description(self):
+    return 'Displays info about channels on Slack.'
+
+  def make_parser(self):
+    parser = ArgumentParser(prog = self.name(), description = self.description())
+    parser.add_argument('-l', '--limit', type = int, default = 50,
+                        help = 'Maximum number of channels to return per page (defaults to 50).')
+    parser.add_argument('-n', '--no-follow', action = 'store_true',
+                        help = 'Don\'t follow cursors.')
```

Comment:
Should we use double quotes where we can avoid escaping characters?

Reply:
No idea. Let's see PEP8.